Conduit/Conduit.Db/Article.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Conduit.Db
+{
+    public class Article
+    {
+        public int ArticleId { get; set; }
```

Comment:
@noor848 why not Guid or long? not recommended to use int as id

Reply:
i did it for user and follower ill  change it for the rest